inst/chunks/fun_is_equal.stan

```diff
@@ -0,0 +1,15 @@
+  // are two 1D integer arrays equal?
+  int is_equal(int[] a, int[] b) {
+    int n_a = size(a);
+    int n_b = size(a);
```

Comment:
`int n_b = size(b);` is correct. Sorry for that.

Reply:
Thanks.